test/libsolidity/SMTCheckerTest.h

```diff
@@ -62,7 +62,8 @@ class SMTCheckerTest: public SyntaxTest
 
 	ModelCheckerSettings m_modelCheckerSettings;
 
-	bool m_ignoreCex = false;
+	// we ignore counterexamples by default
+	bool m_ignoreCex = true;
```

Comment:
Do we actually need to change this?
It does not really belong to this PR, no?

Reply:
I think so. It looks misleading that we set it here to false and then use `yes` value as default when reading the settings.

Reply:
I don't quite agree, because the constructor decides what is the default.
I would at least not put the comment here, because what if we decide to switch the default in the constructor, then this again becomes outdated.

Also, we have similar situation for other settings. For example `showUnsupported` in `ModelCheckerSettings` has initial value `false`, but in the constructor of `SMTCheckerTest` the default is `yes` which sets the field to `true`.

I agree that it is somewhat misleading. Not sure what the best solution is, but I don't think that the initialization at the declaration should be marked as the source of the truth, when we actually decide the default value in the body of the constructor.

Reply:
ok, I'll leave it, like it was before.